src/epub/text/bertie-changes-his-mind.xhtml

```diff
@@ -178,9 +178,9 @@
 			<p>A silvery peal of laughter from the front row stopped him again.</p>
 			<p>“Girls!” said Miss Tomlinson. She spoke in a low, soft voice, but the effect was immediate. Perfect stillness instantly descended upon all present. I am bound to say that, brief as my acquaintance with Miss Tomlinson had been, I could recall few women I had admired more. She had grip.</p>
 			<p>I fancy that Miss Tomlinson had gauged <abbr>Mr.</abbr> Wooster’s oratorical capabilities pretty correctly by this time, and had come to the conclusion that nothing much in the way of a stirring address was to be expected from him.</p>
-			<p>“Perhaps,” she said, “as it is getting late, and he has not very much time to spare, <abbr>Mr.</abbr> Wooster will just give you some little word of advice which may be helpful to you in after-life, and then we will sing the school song and disperse to our evening lessons.”</p>
+			<p>“Perhaps,” she said, “as it is getting late, and he has not very much time to spare, <abbr>Mr.</abbr> Wooster will just give you some little word of advice which may be helpful to you in afterlife, and then we will sing the school song and disperse to our evening lessons.”</p>
```

Comment:
Both of these I left with dashes because he's not referring to THE afterlife, but rather "after life" which I think is a distinction that should be kept. Can you just edit this commit to undo this, but without adding another commit? Thanks!

Reply:
I don't know... can I ? :-). I am looking at how to do it and it seems to imply I have to rebase then amend the commit. Is that what you are thinking?

Reply:
Correct

Reply:
Not sure what I actually did but its fixed. If it isn't what was needed I might have to go back and start again because I got pretty lot in the rebases et al.